dashboard/app/helpers/users_helper.rb

```diff
@@ -91,11 +91,20 @@ def summarize_user_progress_for_all_scripts(user)
           ul = uls.try(:[], level_id)
           completion_status = activity_css_class(ul)
           submitted = !!ul.try(:submitted)
-          if completion_status != 'not_tried'
+          view_answers = !!ul.try(:view_answers)
```

Comment:
Don't need to `.try` here if `:view_answers` is a column on `UserLevel`.  If you're concerned about the nil case, make the column non-nullable.